src/Analyzers/Core/CodeFixes/NamingStyle/NamingStyleCodeFixProvider.cs

```diff
@@ -40,64 +31,53 @@ public NamingStyleCodeFixProvider()
         public override ImmutableArray<string> FixableDiagnosticIds { get; }
             = ImmutableArray.Create(IDEDiagnosticIds.NamingRuleId);
 
-        public override FixAllProvider? GetFixAllProvider()
-        {
-            // Currently Fix All is not supported for naming style violations.
-            return null;
-        }
+        public override FixAllProvider? GetFixAllProvider() => WellKnownFixAllProviders.BatchFixer;
```

Comment:
⚠️  Batch fixer isn't going to work here (it will be brutally slow, to the point that fixing manually is a much better experience)